routers/common/repo.go

```diff
@@ -52,55 +53,62 @@ func ServeData(ctx *context.Context, name string, size int64, reader io.Reader)
 		buf = buf[:n]
 	}
 
-	ctx.Resp.Header().Set("Cache-Control", "public,max-age=86400")
+	httpcache.AddCacheControlToHeader(ctx.Resp.Header(), 5*time.Minute)
 
 	if size >= 0 {
 		ctx.Resp.Header().Set("Content-Length", fmt.Sprintf("%d", size))
 	} else {
-		log.Error("ServeData called to serve data: %s with size < 0: %d", name, size)
+		log.Error("ServeData called to serve data: %s with size < 0: %d", filePath, size)
 	}
-	name = path.Base(name)
-
-	// Google Chrome dislike commas in filenames, so let's change it to a space
-	name = strings.ReplaceAll(name, ",", " ")
 
+	fileName := path.Base(filePath)
 	st := typesniffer.DetectContentType(buf)
```

Comment:
renaming this to something more expressive would help readability

Reply:
You mean the `st` variable? Yes, I agree. Should probably also rename the `cs` but it's kind of in conflict with `modules/charset` 😆

Reply:
rename done of both.